vllm/entrypoints/chat_utils.py

```diff
@@ -888,7 +888,9 @@ def _parse_chat_message_content(
             parsed_msg = _AssistantParser(message)
 
             if "tool_calls" in parsed_msg:
-                result_msg["tool_calls"] = list(parsed_msg["tool_calls"])
+                tool_calls = list(parsed_msg["tool_calls"])
+                if len(tool_calls) > 0:
+                    result_msg["tool_calls"] = tool_calls
```

Comment:
If you wanted to be fancy about it with the Python:

```suggestion
            if len(tool_calls := list(parsed_msg.get("tool_calls", []))) > 0:
                result_msg["tool_calls"] = tool_calls
```